### PR TITLE
feat: add resource org-info commad

### PR DESCRIPTION
**Pull request by LiteSun**
related jira task: https://api7ai.atlassian.net/browse/CLOUD-1550?atlOrigin=eyJpIjoiMWYxMzE4MTM5NzkwNDkzYmE0OGQ3NTZkMGI0NDEyYjciLCJwIjoiaiJ9

image

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -15,6 +15,7 @@
 package main
 
 import (
+	"github.com/api7/cloud-cli/cmd/resource"
```
Comment (nic-6443) [inline main.go:18]: sort imports.

File: cmd/resource/org_info.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+					} else {
+						out, err := json.Marshal(org)
+						if err != nil {
+							output.Warnf("Failed to parse organization info %s", err.Error())
+							return
+						}
+						output.Infof(string(out))
+					}
+				}
+			}
```
Comment (mscb402) [inline cmd/resource/org_info.go:60]: Can we optimize this? This is so deep 😂

Comment (LiteSun) [reply cmd/resource/org_info.go]: updated.

File: cmd/resource/org_info.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+				return
+			}
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			config, err := persistence.LoadConfiguration()
+			if err != nil {
+				output.Errorf(err.Error())
+			}
+
+			for _, profile := range config.Profiles {
```
Comment (tokers) [inline cmd/resource/org_info.go:44]: We should only print the org-info for the default profile.

File: cmd/resource/org_info.go [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+					output.Warnf("Failed to get default organization for profile %s: %s", profile.Name, err.Error())
+					return
+				}
+
+				out, err := json.Marshal(org)
+				if err != nil {
+					output.Warnf("Failed to parse organization info %s", err.Error())
+					return
+				}
+				output.Infof(string(out))
```
Comment (tokers) [inline cmd/resource/org_info.go:66]: Let's not print it via the log message. It may take time information.

**Review by mscb402 — APPROVED** [review]

File: cmd/resource/org_info.go (hunk 3 above)
Comment (LiteSun) [reply cmd/resource/org_info.go:44]: only printing the org-info was handled by the line45 - line 47

File: main.go (hunk 1 above)
Comment (LiteSun) [reply main.go]: updated

File: cmd/resource/org_info.go (hunk 4 above)
Comment (LiteSun) [reply cmd/resource/org_info.go]: updated

**Review by nic-6443 — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**